app/models/proposal.rb

```diff
@@ -59,8 +59,7 @@ class Proposal < ApplicationRecord
   }
 
   scope :submitted_type, lambda { |type|
-    where(status: 1)
```

Comment:
Proposals may have a non-0 status, with an assigned code. This could happen if a proposal was submitted but then changes were requested so the status changes to something other than 1.